.github/workflows/test.yml

```diff
@@ -0,0 +1,26 @@
+name: Tests
+
+on:
+    push:
```

Comment:
@fpena GitHub Actions is set to be triggered by either push/pull-request to master branch.